src/Components/AuthForm.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect } from "react";
```

Comment:
Nice work developing this authForm, I like how you've lifted state from this component to use throughout the application well done.